### PR TITLE
Replace UUID generator for SecureRandom

### PR DESCRIPTION
**Pull request by maratsubkhankulov**

### CODE REVIEW

**Comment by michaelScoff** [comment]
@Krocks can you take a look?

File: app/src/main/java/com/wallet/crypto/trustapp/views/SplashActivity.java [hunk 1]
```diff
@@ -17,9 +18,9 @@
     protected void onCreate(Bundle savedInstanceState) {
 
         super.onCreate(savedInstanceState);
-        if (!BuildConfig.DEBUG) {
-            Fabric.with(this, new Crashlytics());
-        }
+
+        Fabric.with(this, new Crashlytics.Builder()
+                .core(new CrashlyticsCore.Builder().disabled(BuildConfig.DEBUG).build()).build());
```
Comment (michaelScoff) [inline app/src/main/java/com/wallet/crypto/trustapp/views/SplashActivity.java]: is this related to this PR?

Comment (maratsubkhankulov) [reply app/src/main/java/com/wallet/crypto/trustapp/views/SplashActivity.java]: no, please ignore
